parm/config/config.base.emc.dyn

```diff
@@ -165,10 +165,10 @@ export gfs_cyc=@gfs_cyc@ # 0: no GFS cycle, 1: 00Z only, 2: 00Z and 12Z only, 4:
 # GFS output and frequency
 export FHMIN_GFS=0
 
-export FHMAX_GFS_00=384
-export FHMAX_GFS_06=384
-export FHMAX_GFS_12=384
-export FHMAX_GFS_18=384
+export FHMAX_GFS_00=240
```

Comment:
Please also change these back to the "384" default...unless there is a science reason it should be "240". Users can change this in their EXPDIRs for their needs. Thanks!